packages/health-gorilla-core/package.json

```diff
@@ -1,20 +1,9 @@
 {
-  "name": "@medplum/health-gorilla",
-  "version": "3.2.9",
+  "name": "@medplum/health-gorilla-core",
+  "version": "3.2.8",
+  "private": true,
   "description": "Medplum Health Gorilla SDK",
-  "homepage": "https://www.medplum.com/",
```

Comment:
Any objections to keeping these values here?  I think someone told us once that these improve NPM ranking algorithm.